### DISCUSSION TITLE
YAML Parse Error of a column option

### DISCUSSION THREAD

**Discussion by As-Design18** [RafaelGB/obsidian-db-folder#726] [Q&A]
### What happened?

I experienced this error multiply times today while setting up different Tables. 
<img width="707" alt="image" src="https://user-images.githubusercontent.com/117035403/210185812-b3cdcd09-e4f2-49ed-ab9d-41fa64863880.png">
i dont know what i am doing wrong and i recreated my tables 3-4 times and the error came up every single time after 30-50 min. I would appreciate some help. I dont think it is really a bug, because i think its just a user problem but i dont know how to fix it. 

### What platform were you using?

Desktop

### Version of the plugin

newest

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hello @As-Design18 , Could you share your ddbb File? I will check the problem and try to control it in future versions

**Comment by As-Design18** [comment]
[Value Goals.md](https://github.com/RafaelGB/obsidian-db-folder/files/10332865/Value.Goals.md)

**Comment by As-Design18** [comment]
Not sure if this file is correct. I am sorry im not really into this programming thing 😅

**Comment by RafaelGB** [comment]
You have a corrupted label option (not sure why)

Just remove this line inside your file

`- { label: "{"{ VALUE:upcoming, in progress, completed }":null}", backgroundColor: "hsl(165, 95%, 90%)"}`
